components/infieldbutton/stories/infieldbutton.stories.js

```diff
@@ -1,80 +1,78 @@
 import { default as IconStories } from "@spectrum-css/icon/stories/icon.stories.js";
+import { uniqueUiIconBaseNames } from "@spectrum-css/icon/stories/utilities.js";
+import { Sizes, withDownStateDimensionCapture } from "@spectrum-css/preview/decorators";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
-import { isActive, isDisabled, isFocused, isHovered, isQuiet, size } from "@spectrum-css/preview/types";
+import { isActive, isDisabled, isHovered, isQuiet, size } from "@spectrum-css/preview/types";
 import metadata from "../dist/metadata.json";
 import packageJson from "../package.json";
 import { InfieldButtonGroup } from "./infieldbutton.test.js";
-import { Template } from "./template.js";
+import { InfieldButtonGroupVariant, Template } from "./template.js";
 
 /**
- * The in-field button component is a button used inside a text field.
+ * In-field buttons are used to represent actions within input fields. They’re currently used inside the [combo box,](/docs/components-combobox--docs) number field, and [search field.](/docs/components-search--docs)
  */
 export default {
 	title: "In-field button",
 	component: "InFieldButton",
 	argTypes: {
 		size: size(["s", "m", "l", "xl"]),
 		isQuiet,
-		position: {
-			name: "Position",
-			type: { name: "string", required: true },
-			table: {
-				type: { summary: "string" },
-				category: "Component",
-			},
-			options: ["left", "right", "top", "bottom"],
-			control: "select"
-		},
 		iconName: {
-			...IconStories?.argTypes?.iconName ?? {},
+			...IconStories?.argTypes?.uiIconName ?? {},
+			options: uniqueUiIconBaseNames,
 			if: false,
+			description: "All UI icons have sizes of `s`, `m`, `l`, and `xl` except for `ArrowDown`, `ArrowLeft`, `ArrowRight`, and `ArrowUp` which only have sizes of `m`. `Asterisk` has all sizes except for `s`.",
 		},
 		isDisabled,
-		isFocused,
 		isActive,
 		isHovered,
 		isStacked: { table: { disable: true } },
```

Comment:
Oh this is interesting. Do we want to enable this control now? Especially with the new number field/stepper featuring the stacked infield buttons?